src/main/java/ch/njol/skript/effects/EffResourcePackValues.java

```diff
@@ -0,0 +1,98 @@
+package ch.njol.skript.effects;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Effect;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.sections.EffSecSendResourcePack.ResourcePackEvent;
+import ch.njol.util.Kleenean;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+import org.jetbrains.annotations.UnknownNullability;
+
+@Name("Resource Pack Values")
+@Description({
+	"Optional values to be used when sending a resource pack",
+	"Can only be used in a Resource Pack Effect Section",
+	"",
+	"The uuid is used to store and indetify the resource pack. Use this when you want to remove a specific resource pack.",
+	"You can allow up to 32 alphanumeric characters in the uuid.",
+	"",
+	"The hash is used for caching, the player won't have to re-download the resource pack that way. ",
+	"The hash must be SHA-1, you can get the SHA-1 hash of your resource pack using ",
+	"[**this online tool**](https://emn178.github.io/online-tools/sha1_checksum.html)",
```

Comment:
```suggestion
	"The hash is used for caching, so that the player doesn't have to re-download the pack each time they join.",
	"The hash must be SHA-1, you can get the SHA-1 hash of your resource pack using " +
	"[**this online tool**](https://emn178.github.io/online-tools/sha1_checksum.html)",
```